hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMDbCheckpointServlet.java

```diff
@@ -244,16 +236,31 @@ private void setupCluster() throws Exception {
         anyBoolean());
   }
 
-  @ParameterizedTest
-  @MethodSource("getHttpMethods")
-  public void testEndpoint(String httpMethod) throws Exception {
-    this.method = httpMethod;
-
+  @Test
+  void testWithoutACL() throws Exception {
     conf.setBoolean(OZONE_ACL_ENABLED, false);
     conf.set(OZONE_ADMINISTRATORS, OZONE_ADMINISTRATORS_WILDCARD);
 
     setupCluster();
 
+    testBootstrapLocking();
+
+    testEndpoint("POST");
+    testEndpoint("GET");
+    testDoPostWithInvalidContentType();
+
+    prepSnapshotData();
+
+    testWriteDbDataWithoutOmSnapshot();
+    testWriteDbDataToStream();
+    testWriteDbDataWithToExcludeFileList();
+  }
+
+  private void testEndpoint(String httpMethod) throws Exception {
+    this.method = httpMethod;
+
+    setupMocks();
+
     final OzoneManager om = cluster.getOzoneManager();
```

Comment:
I think we can add om as a class attribute while setting up the cluster, since it is used several times.